frontend/simple/views/Mailbox.vue

```diff
@@ -0,0 +1,284 @@
+                <div><strong>From:</strong>&nbsp;{{currentMessage.groupId ?  currentMessage.groupId : currentMessage.from}}</div>
+              </div>
+              <p class="panel-block" v-if="currentMessage.message" style="display: block; word-wrap: break-word;">{{currentMessage.message}}</p>
+              <p class="panel-block" v-if="currentMessage.groupId"><router-link v-bind:to="{ path: '/join', query: { groupId: currentMessage.groupId} }" ><i18n>Respond to Invite</i18n></router-link></p>
+              <div class="panel-block" >
+                <button class="button is-danger" v-if="!currentMessage.groupId" type="submit" style="margin-left:auto; margin-right: 0" v-on:click="remove(index)"><i18n>Delete</i18n></button>
+                <button class="button is-primary" type="submit" v-on:click="inboxMode" style="margin-left:10px; margin-right: 0"><i18n>Return</i18n></button>
+              </div>
+            </div>
+            <table id="Invites" class="table is-bordered is-striped is-narrow"  v-show="(mode === 'Inbox')" v-if="invites.length">
+              <thead>
+              <tr>
+                <th><i18n>Invites</i18n></th>
+              </tr>
+              </thead>
+              <tbody>
+              <tr v-for="(message, index) in invites">
+                <td>
+                  <div class="media">
+                    <div class="media-left" v-on:click="readInvite(index)">
+                      <p class="image is-64x64">
+                        <img src="http://bulma.io/images/placeholders/128x128.png">
+                      </p>
+                    </div>
+                    <div class="media-content invite-message" v-on:click="readInvite(index)">
+                      <div><strong>Sent:</strong>&nbsp;{{formatDate(message.sentDate)}}</div>
+                      <div><strong>From:</strong>&nbsp;{{message.groupId}}</div>
+                    </div>
+                  </div>
+                </td>
+              </tr>
+              </tbody>
+            </table>
+            <table id="Inbox" class="table is-bordered is-striped is-narrow" v-show="(mode === 'Inbox')">
+              <thead>
+              <tr>
+                <th><i18n>Inbox</i18n></th>
+              </tr>
+              </thead>
+              <tbody>
+              <tr v-for="(message, index) in inbox">
+                <td>
+                  <div class="media">
+                    <div class="media-left" v-on:click="read(index)">
+                      <p class="image is-64x64">
+                        <img src="http://bulma.io/images/placeholders/128x128.png">
+                      </p>
+                    </div>
+                    <div class="media-content inbox-message" v-on:click="read(index)">
+                      <div><strong>Sent:</strong>&nbsp;{{formatDate(message.sentDate)}}</div>
+                      <div><strong>From:</strong>&nbsp;{{message.from}}</div>
```

Comment:
Missing `<i18n>`'s here and in a bunch of other places. Look through all .vue files in this PR to make sure they're there.